### PR TITLE
Additional Kotlin Improvements 

### PR DESCRIPTION
**Pull request by iNoles**
1) Upgrade Kotlin to 1.8.10
2) Implement Parameters

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -1,11 +1,9 @@
 plugins {
-    kotlin("multiplatform") version "1.7.20" apply false
-    id("org.jetbrains.kotlinx.kover") version "0.6.1" apply false
+    kotlin("multiplatform") version "1.8.10" apply false
+    id("org.jetbrains.kotlinx.kover") version "0.7.0-Alpha" apply false
```
Comment (kittinunf) [inline build.gradle.kts:3]: Shall we use kover alpha? or it is not compatible with new Kotlin version?

Comment (iNoles) [reply build.gradle.kts:3]: well, it is the latest version for Kover.

Comment (kittinunf) [reply build.gradle.kts:3]: yeah, what I meant was that maybe we should stop using them until they get a bit more mature.

File: fuel/src/commonMain/kotlin/fuel/Request.kt [hunk 2]
```diff
@@ -1,7 +1,10 @@
 package fuel
 
+public typealias Parameters = List<Pair<String, String>>
```
Comment (kittinunf) [inline fuel/src/commonMain/kotlin/fuel/Request.kt:3]: I'm wondering whether we should do `Pair<String, Any?>` for this.

File: build.gradle.kts (hunk 1 above)
Comment (iNoles) [reply build.gradle.kts:3]: Oh Okay, I see what you mean. Jacoco is not that multiplatform friendly either.

**Comment by kittinunf** [comment]
Looks good for me sir! Just wondering whether we should support something rather than `Pair<String, String>`

**Review by kittinunf — APPROVED** [review]
🎆

**Pull request merged**